R/estimateEvenness.R

```diff
@@ -114,28 +114,25 @@
 #'
 #' # Estimate evenness and give polished names to be used in the output
 #' tse <- estimateEvenness(tse, index = index, name = name)
-#'
 #' # Check the output
 #' head(colData(tse))
 #'
 NULL
 
-#' @rdname estimateEvenness
-#' @export
-setGeneric("estimateEvenness",signature = c("x"),
-           function(x, assay.type = assay_name, assay_name = "counts",
-                    index = c("pielou", "camargo", "simpson_evenness", "evar",
-                              "bulla"),
-                    name = index, ...)
-               standardGeneric("estimateEvenness"))
+setGeneric(
+    ".estimate_evenness",signature = c("x"),
+    function(
+        x, assay.type = assay_name, assay_name = "counts",
+        index = c("pielou", "camargo", "simpson_evenness", "evar", "bulla"),
+        name = index, ...)
+        standardGeneric(".estimate_evenness"))
```

Comment:
is there an actual change here? if not, then it would be better to keep the original format so we can focus the review on the actual changes. Same comment goes for all content.

Reply:
yes: estimate_evenness --> .estimate_evenness